azure-pipelines.yml

```diff
@@ -24,6 +28,19 @@ trigger: none
 
 name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
 
+resources:
+  repositories:
+    - repository: sonic-mgmt
+      type: github
+      name: sonic-net/sonic-mgmt
+      ref: master
+      endpoint: sonic-net
+
+parameters:
+- name: TIMEOUT_IN_MINUTES_PR_TEST
```

Comment:
Currently we have 2 timeout settings,
one is in pipeline, another one is in test plan itself.
I'm thinking, we can use a wide constraints for pipelie, for example 12 hours.
And use a runtime variable to be input as test plan time out(e.g. 4 hours, 6 hours on demand).
So that it's controllable.

Reply:
yeah, we can start with 8 hours